docs/en/reference/sql/task_manage/SHOW_JOBS.md

```diff
@@ -1,16 +1,18 @@
 # SHOW JOBS
 
+The `SHOW JOBS` statement displays the list of tasks that have been submitted in offline or online mode of the cluster version.
```

Comment:
online mode only `LOAD DATA`, others won't be shown in here.
is it better?
```
submit to taskmanager, all offline job, online `LOAD DATA` job
```

Reply:
Is it ok if I change it to :

The `SHOW JOBS` statement displays the list of tasks that have been submitted to taskmanager in the cluster version, including all kinds of jobs in offline mode and `LOAD DATA` jobs in online mode